### PR TITLE
lazy loading and width+height for <img>

### PR DESCRIPTION
**Pull request by midzer**
Hi,

been a pleasure to work on this :)

* reorder `<head>`, check https://github.com/rviscomi/capo.js for reference
* preload hero font
* slightly larger `odd-logo.svg`
* `loading="lazy"` for all below the fold images
* width+height for all images to reduce Layout Shift, check https://web.dev/articles/optimize-cls
* ~~slightly larger Partner images~~
* ~~moved own logo to bottom ("the donkey calls itself at the end")~~

Further possible optimizations regarding page payload:

* reduce .svg size with https://jakearchibald.github.io/svgomg/ (our own logo would save 50%)
* hero background to .svg (perhaps original file was a .svg anyway?)

Cya on monday,
midzer

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -3,9 +3,12 @@
 <head>
     <meta charset="utf-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <link rel="shortcut icon" type="image/x-icon" href="./static/favicon.ico">
     <title>Open Data Day Flensburg - 2. März 2024</title>
+    <link rel="stylesheet" href="./main.css">
+    <link rel="preload" href="./static/fonts/grotesk/HKGrotesk-Bold.woff2" as="font" type="font/woff2" crossorigin>
+    
     <meta name="description" content="Open Data Day 2024 in Flensburg - Treffen Sie Experten, tauschen Sie Ideen aus und helfen Sie mit, die Digitalisierung demokratisch zu gestalten.">
+    <link rel="shortcut icon" type="image/x-icon" href="./static/favicon.ico">
```
Comment (p3t3r67x0) [inline src/index.html:11]: passt

File: src/index.html [hunk 2]
```diff
@@ -3,9 +3,12 @@
 <head>
     <meta charset="utf-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <link rel="shortcut icon" type="image/x-icon" href="./static/favicon.ico">
     <title>Open Data Day Flensburg - 2. März 2024</title>
+    <link rel="stylesheet" href="./main.css">
```
Comment (p3t3r67x0) [inline src/index.html:7]: passt

File: src/index.html [hunk 3]
```diff
@@ -3,9 +3,12 @@
...
+    <link rel="preload" href="./static/fonts/grotesk/HKGrotesk-Bold.woff2" as="font" type="font/woff2" crossorigin>
```
Comment (p3t3r67x0) [inline src/index.html:8]: passt

File: src/index.html [hunk 4]
```diff
@@ -315,58 +312,66 @@ <h3 class="text-2xl lg:text-5xl font-bold mb-4 lg:mb-8">Komm vorbei auf 'nen Kaf
...
                 </div>
                 <div class="col-span-12 lg:col-span-6">
                     <a href="https://www.schleswig-holstein.de/DE/landesregierung/ministerien-behoerden/I/Staatskanzlei/staatskanzlei_node.html" class="flex items-center justify-center h-14 md:h-32" title="Staatskanzlei Schleswig-Holstein" target="_blank">
-                        <img src="./static/echtdigital_staatskanzlei.svg" class="w-3/5" alt="Staatskanzlei Schleswig-Holstein">
+                        <img src="./static/echtdigital_staatskanzlei.svg" class="w-4/5" alt="Staatskanzlei Schleswig-Holstein" loading="lazy" width="300" height="62">
                     </a>
                 </div>
             </div>
         </div>
+
```
Comment (p3t3r67x0) [inline src/index.html]: Thx, for your suggestion, we must discuss design changes first with @roaldchristesen and @chanty1512

File: src/index.html [hunk 5]
```diff
@@ -315,58 +312,66 @@ <h3 class="text-2xl lg:text-5xl font-bold mb-4 lg:mb-8">Komm vorbei auf 'nen Kaf
     <div class="bg-white">
         <div class="container mx-auto px-3 xl:px-0">
             <h3 class="text-2xl lg:text-5xl font-bold pt-3 mb-4 lg:py-8">Partner</h3>
-            <p class="text-lg lg:text-2xl leading-7 mb-5 lg:mb-8">Ein Event vom OK Lab Flensburg. Mit freundlicher Unterstützung von unseren Partnern und Förderern</p>
```
Comment (p3t3r67x0) [inline src/index.html]: Thx. @roaldchristesen would you be so kind and have a look? As far as I know the wording is the same as on our print layouts. Could you be so kind and join this discussion?

**Review by p3t3r67x0 — CHANGES_REQUESTED** [review]
@midzer thank you for your pull request. Would you mind, unfortunately we couldn't approve your PR directly due to some minor issues. Images with a fixed width and height does not look correct in the viewports XS, SM, MD, LG and XL. These images does not retain the desired optical shape in all viewports in all tested devices. Further more, we cannot change logo width and height in the partner section, because any changes must be discussed in advance with the partners and the arrangement of the logos in the respective viewports. The arrangement in the respective viewports have already been approved. The idea preloading fonts is pretty great. The order of the `<head>` makes sense and looks good. Using attribute `loading="lazy"` is pretty straight forward. Best regards

File: src/index.html (hunk 5 above)
Comment (roaldchristesen) [reply src/index.html]: @midzer Danke für deine Optimierungsvorschläge!
Da wir in allen unseren anderen Materialien den Satz "Ein Event vom OK Lab Flensburg. Mit freundlicher Unterstützung von unseren Partnern und Förderern" verwenden, würde ich ihn für dieses Jahr gerne so belassen.
Ebenfalls das Logo oben. So kurz vor dem Event möchten wir aus Kapizitätsgründen keine Designänderungen vornehmen.

Comment (midzer) [reply src/index.html]: Okay, ich reverte genannte Änderungen später.

**Review by p3t3r67x0 — APPROVED** [review]
@midzer vielen dank für dein tollen Input. :rocket:

**Pull request merged**